test/e2e/systemd_activate_test.go

```diff
@@ -64,16 +66,34 @@ var _ = Describe("Systemd activate", func() {
 		Expect(err).ToNot(HaveOccurred())
 		addr := net.JoinHostPort(host, strconv.Itoa(port))
 
+		// Make a temporary root directory
+		tmpRootDir := filepath.Join(tempDir, "server_root")
+		err = os.Mkdir(tmpRootDir, 0755)
+		Expect(err).ToNot(HaveOccurred())
+		defer os.RemoveAll(tmpRootDir)
+
+		// When SELinux is enabled, a storage root directory should be
+		// labeld with a specific value
+		if selinux.GetEnabled() {
+			rootDir := "/var/lib/containers"
+			if rootless.IsRootless() {
+				rootDir = filepath.Join(os.Getenv("HOME"), ".local/share/containers")
```

Comment:
This makes me nervous: a little because of the hardcoding, but more so because it requires the existence of that user directory, which requires that the user have run `podman` at least once without `--root` / `--runroot` options. Do we have that guarantee in e2e tests? CI passed, so I guess it works in practice, ...?

Reply:
Good point, could you run a test to see if the directory exists before the chcon.

Reply:
I fixed it.
If rootDir doesn't exist, use "chcon -t" to label tmpRootDir instead of "chcon --reference".